src/base/static/components/place-detail/index.js

```diff
@@ -185,6 +185,24 @@ class PlaceDetail extends Component {
     }
   }
 
+  onAttachmentModelRemove(attrs, modelId) {
+    if (confirm(this.props.t("confirmAttachmentRemove"))) {
```

Comment:
An undo option would be better than the `confirm` UI here, but I'd say we need to think through how undo will work first. Eventually with most of the state in Redux I think undo becomes a lot easier to implement, because we can just roll back the Redux state a step at a time.